Makefile

```diff
@@ -239,6 +268,7 @@ conversion-gen: $(CONVERSION_GEN) ## Build a local copy of conversion-gen.
 conversion-verifier: $(CONVERSION_VERIFIER) ## Build a local copy of conversion-verifier.
 gotestsum: $(GOTESTSUM) ## Build a local copy of gotestsum.
 yq: $(YQ)  ## Build a local copy of yq.
+go-apidiff: $(GO_APIDIFF) ## Build a local copy of go-apidiff
```

Comment:
this (and all similar targets above) could be:

.PHONY: $(GO_APIDIFF_BIN)
$(GO_APIDIFF_BIN): $(GO_APIDIFF) ## Build a local copy of go-apidiff